presto-product-tests/conf/docker/singlenode-mysql/docker-compose.yml

```diff
@@ -4,10 +4,6 @@ services:
   mysql:
     hostname: mysql
     image: 'mysql:5.7'
-    ports:
-      - '13306:13306'
```

Comment:
The reason for using non-default port was that we can expose & bind it on localhost, without conflicting with a locally running service. It doesn't seem important for me (i dont run mysql locally). After your change it seems impossible to run mysql product tests directly from intellij.

Reply:
But I removed the port exposure. In case we want to expose them we should use random host port.

Reply:
BTW do you know how to run convention tests from IDE? MySQL has only one test that is not convention test. For manual testing I would use MySQL query runner from IDE.